README.md

```diff
@@ -1,17 +1,39 @@
-# SOEN341-InsertName
+﻿# SOEN341-InsertName
 ### Authors
-* Simon Lim (tech lead) | 
-* Émilie Martin | @ emilie-martin
-* Daniela Venuta | @ daniela-venuta
-* Minh-Tam Do |
-* Luc Nguyen |
-* Karin Kazarian |
+* Simon Lim (tech lead) | @nasaku898
+* Émilie Martin | @emilie-martin
+* Daniela Venuta | @daniela-venuta
+* Minh-Tam Do | @mint47
+* Luc Nguyen | @1999lucnguyen
+* Karin Kazarian | @Karin-kazarian
+
+### Description
+This is a project partial fulfillement of the requirements for the course SOEN341 at Concordia University.
+### Objectives
+The app is a replication of the social media Instagram from which we will add on features.
+### Core Features
+* User Service (Login and account management)
+* Posting Pictures
+* Reacting (comment & likes)
+* Following other users
+* (other) tba
+
+[More about our core features](https://github.com/emilie-martin/SOEN341-InsertName/wiki/Program-Breakdown)
 
 ### Technology Used
+This app uses the following technologies :
```

Comment:
Maybe specifies which programming language is being used. In our case, Java for the back-end and Javascript for front-end.

Reply:
To add to this, maybe we could make two lists: Front-end and Back-end and then have nested lists for the technologies we use for each aspect of the project?